reference/OpenAPI.json

```diff
@@ -24245,93 +24245,247 @@
-                "items": {
-                  "type": "string"
-                }
-              },
-              "filters": {
-                "description": "Data filters used to narrow the data records to be returned in the export.",
-                "oneOf": [
-                  {
-                    "$ref": "#/components/schemas/8_req_filter_campaign_id"
+              "conditions": {
+                "type": "object",
+                "description": "Data filters used to narrow the data records to be returned in the result.",
+                "properties": {
+                  "$in": {
+                    "type": "array",
+                    "description": "Array of resource values that should be included in the results.",
+                    "items": {
+                      "type": "string"
+                    }
                   },
-                  {
-                    "$ref": "#/components/schemas/8_req_filter_voucher_id"
+                  "$not_in": {
+                    "type": "array",
+                    "description": "Array of resource values that should be included in the results.",
+                    "items": {
+                      "type": "string"
+                    }
+                  },
+                  "$is": {
+                    "type": "string",
+                    "description": "Specify the exact resource value."
+                  },
+                  "$is_not": {
+                    "type": "string",
+                    "description": "Results should omit this value."
                   }
-                ]
+                }
               }
             }
           },
-          "result": {
+          "campaign_id": {
             "type": "object",
-            "description": "Contains the URL of the CSV file.",
+            "description": "Data filters used to narrow the data records to be returned in the result.",
             "properties": {
-              "url": {
-                "type": "string",
-                "description": "URL of the CSV file location."
+              "conditions": {
+                "type": "object",
+                "description": "Data filters used to narrow the data records to be returned in the result.",
+                "properties": {
+                  "$in": {
+                    "type": "array",
+                    "description": "Array of resource values that should be included in the results.",
+                    "items": {
+                      "type": "string"
+                    }
+                  },
+                  "$not_in": {
+                    "type": "array",
+                    "description": "Array of resource values that should be included in the results.",
```

Comment:
`"description": "Array of the resource values that should be excluded from the results.",`